pipeline/ip_metadata_values.py

```diff
@@ -0,0 +1,51 @@
+"""A module to hold a bunch of values needed to init an IPMetadataChooser."""
```

Comment:
This file can be deleted. Pass a factory function to the pipeline or make the chooser know how to initialize itself.

Reply:
done